tests/qml/Properties.qml.json

```diff
@@ -131,12 +131,12 @@
           "stat",
           [
             "qmlelem",
-            "Item",
+            "QQ.Item",
```

Comment:
This is not the proper format for `qmlelem`. See [ImportsNamed.qml.json#L27-L32](https://github.com/qmlweb/qmlweb-parser/blob/master/tests/qml/ImportsNamed.qml.json#L27-L32) for how qualified `qmlelem` should be encoded. You could also test with longer paths. e.g. `Abc.Def.Rectangle`.